debezium-connector-sqlserver/pom.xml

```diff
@@ -125,7 +125,7 @@
                                     <color>yellow</color>
                                 </log>
                                 <wait>
-                                    <time>30000</time> <!-- 30 seconds max -->
+                                    <time>60000</time> <!-- 60 seconds max -->
```

Comment:
Also I have to increase this because it seems that new containerized SQLServer takes a little bit more time to start up